packages/react/src/exampleTypes/StepSlider.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { FC, useState } from 'react'
+import _ from 'lodash/fp.js'
+import {
+  RangeSlider,
+  RangeSliderTrack,
+  RangeSliderFilledTrack,
+  RangeSliderThumb,
+  Box,
+  Tooltip,
+  Text,
+  HStack,
+  Stack,
+} from '@chakra-ui/react'
+import { contexturify } from '../utils/hoc'
+import { toNumber } from '../utils/format'
+import { FormatterFn, Node, Tree } from '../types/contexture'
+
+/** This theme is an initial step towards looking further
+ * at the tokenization to be set by our design system. */
+const theme = {
+  color: {
+    blue: { 100: '#CCE7FF', 500: '#0076DE', 700: '#005199' },
+    gray: { 50: '#FFFFFF', 100: '#E7EAEE' },
+    container: { border: { sm: '#E7EAEE' } },
+    text: {
+      body: {
+        primary: '#495569',
+        tertiary: '#697996',
+      },
+    },
+  },
+  radius: { base: { xs: '8px' } },
+  font: { size: { xs: '10px', md: '14px' } },
+}
+
+export type UnknownRec = Record<string, unknown>
+
+type NodeProps = {
+  range: number[]
+  steps: number[]
+}
+
+type ContextProps = {
+  recordsCount: number
+}
+
+interface StepSliderProps {
+  node: Node<NodeProps, ContextProps>
+  formatter: FormatterFn
+  path: string
+  tree: Tree<NodeProps>
+}
+
+const getDefaultRange = (steps: number[], range: number[]): number[] => {
+  if (range.length === 0) return [0, steps.length - 1]
+  const minStep = steps.findIndex((step: number) => step === range[0])
+  const maxStep = steps.findIndex((step: number) => step === range[1])
+  return [minStep, maxStep]
+}
+
+const getRealValue = (steps: number[], value: number): number => {
```

Comment:
A request to elaborate on this name.

Reply:
Right, the "real value" isn't the active or the current ones. What happens is, this Range Slider component is being adapted to become a Step Slider. In order to make that work, we have to "translate" "step" values to its corresponding "real" values. To be more clear:
- Step 0 is 1,000,
- Step 1 is 5,000,
- Step 2 is 10,000
...
- Step 6 is 1,000,000

I would enhance the name but I'm very open to suggestions as well.